libc/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ elseif (OE_TRUSTZONE)
   add_enclave_library(
     oelibasm
     OBJECT
+    ${MUSLSRC}/math/sqrtl.c
```

Comment:
Have you check if newly added source files are covered by our existing libc tests?

Reply:
This I am going to do as a separate PR and update tests/libc/. This is one of the sub-tasks in https://github.com/openenclave/openenclave/issues/3933